docs/development/code_guide.rst

```diff
@@ -344,7 +344,7 @@ takes care of executing them at documentation build time and including them
 in the documentation.
 
 Please note that it is necessary to store notebooks with their outputs stripped
-(use the "Edit -> Clear all outputs" option). This accomplishes two goals:
+(use the "Cell -> All Output -> Clear" option). This accomplishes two goals:
```

Comment:
Ah, I see what's up in here! Let's reword this:

```suggestion
(use the "Edit -> Clear all" option in JupyterLab and the "Cell -> All Output -> Clear" option in the "classic" Jupyter Notebook). This accomplishes two goals:
```

We probably wrote this with JupyterLab already in mind, and didn't think to double check. Thanks for pointing this out!